projects/packages/search/package.json

```diff
@@ -0,0 +1,72 @@
+{
+	"name": "jetpack-search",
+	"version": "0.2.0-alpha",
+	"description": "Package for Jetpack Search products",
+	"main": "main.js",
+	"directories": {
+		"test": "tests"
+	},
+	"scripts": {
+		"clean-all": "rm -rf build/* .cache/* node_modules/* node_modules/.[!.]* vendor/*",
+		"build": "echo 'Not yet implemented.'",
+		"test": "pnpx concurrently 'pnpm:test-scripts' 'pnpm:test-size'",
+		"test-scripts": "jest",
```

Comment:
As there are no tests yet, shall we add `--passWithNoTests` so this succeeds?

Reply:
good point. addressed in 55d208ab39